profiles/coreos/base/package.accept_keywords

```diff
@@ -12,8 +12,8 @@
 =dev-lang/rust-1.57.0-r1 ~amd64 ~arm64
 =virtual/rust-1.57.0 ~amd64 ~arm64
 
-# dev-libs/openssl-3.0.0 is still in testing phase at this point
-=dev-libs/openssl-3.0.0 ~amd64 ~arm64
+# dev-libs/openssl-3.0.1 is still in testing phase at this point
```

Comment:
I suppose that since openssl is in overlay, we could mark it as stable in the ebuild. What do you think?

Reply:
Most of the time, I try to keep drift between `::gentoo` and `::coreos-overlay` as minimal as possible. Even to mark a package as stable.

Now you say it, if we update directly the ebuild, we could decrease the number of file to modified when we update a package (`package.accept_keywords` in this case) and so the mistake probability.

On the other hand, in the documentation[^1], we explicitely mention `package.accept_keywords` file update:
> To proceed, add the package name and version, and its masked architectures to the package.accept_keywords file inside the coreos profile. 


[^1]: https://www.flatcar.org/docs/latest/reference/developer-guides/sdk-modifying-flatcar/#add-or-update-a-package

Reply:
> Most of the time, I try to keep drift between `::gentoo` and `::coreos-overlay` as minimal as possible. Even to mark a package as stable.
> 
> Now you say it, if we update directly the ebuild, we could decrease the number of file to modified when we update a package (`package.accept_keywords` in this case) and so the mistake probability.
> 

Yeah. It's a tradeoff between minimizing the diff, but having changes in couple places vs having a bigger diff, but limited to a single location.

But yeah, I wasn't sure, so I asked for your opinion.


> On the other hand, in the documentation[1](#user-content-fn-1-c098c53cc832f1db184b6d2c38340798), we explicitely mention `package.accept_keywords` file update:
> 
> > To proceed, add the package name and version, and its masked architectures to the package.accept_keywords file inside the coreos profile.

I'd say it this is what we want to do for portage-stable packages (we shouldn't be modifying them, of course).

> 
> ## Footnotes
> 
>     1. https://www.flatcar.org/docs/latest/reference/developer-guides/sdk-modifying-flatcar/#add-or-update-a-package [leftwards_arrow_with_hook](#user-content-fnref-1-c098c53cc832f1db184b6d2c38340798)

Reply:
Ok - then let's give a try by updating directly the ebuild to mark it as stable, it's consistent with the fact of cleaning the ebuild too.
`package.accept_keywords` should only be used for `::portage-stable` ebuilds.